statsmodels/tools/decorators.py

```diff
@@ -4,7 +4,26 @@
 from statsmodels.compat.python import get_function_name
 import warnings
 
-__all__ = ['resettable_cache', 'cache_readonly', 'cache_writable']
+__all__ = ['resettable_cache', 'cache_readonly', 'cache_writable',
+           'deprecated_alias']
+
+
+def deprecated_alias(old_name, new_name, remove_version=None):
+    msg = '%s is a deprecated alias for %s' % (old_name, new_name)
```

Comment:
Would a short docstring with an example help others find/use this?

Reply:
Good idea, will implement.